libbeat/dashboards/config.go

```diff
@@ -11,11 +13,21 @@ type Config struct {
 	OnlyDashboards bool   `config:"only_dashboards"`
 	OnlyIndex      bool   `config:"only_index"`
 	AlwaysKibana   bool   `config:"always_kibana"`
+	Retry          *Retry `config:"retry"`
+}
+
+type Retry struct {
+	Enabled  bool          `config:"enabled"`
+	Interval time.Duration `config:"interval"`
+	Maximum  uint          `config:"maximum"`
 }
 
 var defaultConfig = Config{
 	KibanaIndex:  ".kibana",
 	AlwaysKibana: false,
+	Retry: &Retry{
+		Enabled: false,
```

Comment:
Can you add the defaults for interval and maximum here too?

Reply:
sure thing, what do you think is reasonable? 
1 sec/ unlimited?

Reply:
SGTM, especially as the feature is off by default.

Reply:
done 3bd3affaa9242bd7072207a911f1313ead19c30e